projects/ci-cd.md

```diff
@@ -0,0 +1,67 @@
+# CI/CD Observability Semantic Conventions Working Group
+
+## Description
+
+The goal of this working group is define the semantic conventions for CI/CD Observability and outline 
+a path forward for the Open Telemetry community. 
+
+### Why
+
+The goal of this project is to establish standard semantic conventions for Continuous
+Integration (CI) and Continuous (CD) observability. This will provide a common language
+and standardized formats for CI/CD observability, enabling the community to observe CI/CD systems.
+
+This will broaden the target audience of Open Telemetry to Release Engineering and DevOps teams,
+further cementing Open Telemetry as the industry standard Observability framework.
+
+The timing is ripe to start now. The [CI/CD Observability OTEP](https://github.com/open-telemetry/oteps/pull/223/files)
+has been open since January of 2023 and with the recent changes to the OTEP process, the Kubecon talk, and vendor acknowledgements, there's momentum available to carry this forward. The industry 
+is heavily looking for solutions and watching the related OTEP with interest.
+
+## Deliverables
+
+The CI/CD observability working group is planning to deliver the [CI/CD Observability OTEP](https://github.com/open-telemetry/oteps/pull/223/files) with prototypes in at least two major vendors:
+
+* [GitHub](https://github.com)
+* [GitLab](https://gitlab.com)
+
+A related OTEP that is of great interest to the CI/CD OTEP is that of [Environment Variables as Context Propagators](https://github.com/open-telemetry/opentelemetry-specification/issues/740) which this working group would be available to help with.
+
+> Note: The environment variables as context propagators enables batch systems as a whole, and not just CI/CD systems. We're here to support and will leverage, but it's possible that other working groups already have that OTEP on their radar.
+
+Working prototypes are partially available in these two systems, but have been down outside of the context of this semantic conventions. Thus these prototypes would be updated based on semantic conventions defined.
```

Comment:
Can you link to those prototypes?

Reply:
Perhaps we can organise via a table of sorts?

| Type | Issue | Prototype |
| ----- | ----- | ---------- |
| GitHub | https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27460 | https://github.com/krzko/opentelemetry-collector-contrib/tree/feat-add-githubactionseventreceiver/receiver/githubactionsreceiver |

Have added my GitHub Actions Receiver as reference.

Reply:
I like the idea of a table, absolutely could link the current prototypes that could be leveraged to be brought further along as the spec gets built out.

Reply:
I think I covered most the ones I know about in the table. The GitLab runners prototype might not be public. I'll work behind the scenes on that front.